libs/components/modals/src/lib/modules/modal/modal.component.spec.ts

```diff
@@ -78,6 +79,7 @@ describe('Modal component', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [SkyModalFixturesModule],
+      providers: [SkyResizeObserverMediaQueryService],
```

Comment:
Should we just place this provider in the `SkyModalFixturesModule`? That is where all the other setup lives.

Reply:
No longer applies.